netbench/cdk/src/main/java/com/aws/ClientServerStack.java

```diff
@@ -0,0 +1,115 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.aws;
+
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.services.ecs.Cluster;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ecs.*;
+import software.amazon.awscdk.services.autoscaling.AutoScalingGroup;
+import software.amazon.awscdk.services.ec2.InstanceType;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpoint;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpointOptions;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpointAwsService;
+import software.amazon.awscdk.services.ec2.BastionHostLinux;
+import software.amazon.awscdk.services.ec2.*;
+import software.amazon.awscdk.services.ssm.StringParameter;
+
+
+import software.amazon.awscdk.services.apigateway.LambdaRestApi;
+import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.Function;
+import software.amazon.awscdk.services.lambda.Runtime;
+import software.amazon.awscdk.RemovalPolicy;
+
+import java.util.HashMap;
+
+
+public class ClientServerStack extends Stack {
+    private String cidr;
+    private Vpc vpc;
+
+    public ClientServerStack(final Construct parent, final String id) {
+        this(parent, id, null);
+    }
+
+    public ClientServerStack(final Construct parent, final String id, final ClientServerStackProps props) {
+        super(parent, id, props);
+
+        String instanceType = props.getInstanceType();
+        String stackType  = props.getStackType();
+        this.cidr = props.getCidr();
+
+        this.vpc = Vpc.Builder.create(this, stackType + "-vpc")
```

Comment:
Do these strings (suffix of `-vpc`,  `-vpc-id`, `-cidr`, using `vpc-sec-group`) have to be exactly these values? Are they referenced in any documentation you could link to?

Reply:
These are just names of the constructs defined within the scope of my app. I picked those names as they were relatively descriptive of those constructs to me.

Reply:
Ok, maybe add a comment stating that. I'm just thinking in terms of maintenance of this code, its helpful to know if these are specific strings that cannot change, or they are just for descriptive purposes